release/__init__.py

```diff
@@ -25,9 +25,13 @@
 import pkgpanda.build
 import pkgpanda.util
 import release.storage
-from pkgpanda.util import logger
+from pkgpanda.constants import PACKAGES_DIR
+from pkgpanda.util import is_windows, logger
 
-provider_names = ['aws', 'azure', 'bash']
+if is_windows:
+    provider_names = ['azure', 'bash']
+else:
+    provider_names = ['aws', 'azure', 'bash']
```

Comment:
Is this necessary? Needs a comment at least.

Reply:
Windows is a work in progress and initially we need azure. We will add aws as we move forwards.